Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openjdk:8-jre-alpine
+
+# Set working directory
+WORKDIR /app
+
+# Copy jar file into container
+COPY ./SNAPSHOT.jar /app
+
+# port to expose
+EXPOSE 80
```

Comment:
Do we need to expose this particular port. What port(s) is the application listening on?

Reply:
I just chose one arbitrarily, I don't know which port we'll actually need

Reply:
Well, there's a port that Spring listens on by default, and I don't think it's this one...

Reply:
Hey @hWorblehat, not sure if this is relevant to the conversation above but in our application the default Spring port for the http server is 8080

Reply:
Yep, that's very relevant ;)